.github/workflows/draft-release.yaml

```diff
@@ -19,28 +20,64 @@
 
 
 ---
-name: "Draft new release"
+name: "Draft Release"
+run-name: "Draft Release ${{ inputs.version }} from ${{ github.ref_name }}"
 
 on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'The version you want to release.'
+        description: 'The version you want to release. Ref should be either main for latest releases or a tag for bugfixes. '
         required: true
 
 jobs:
+  validate-release:
+    name: "Validate if version can be released"
```

Comment:
nit: I would put a more explicit description like "Validates that tag <TAG> does not already exist"